File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int k;
        Set<Integer> set = new LinkedHashSet<>();
        for(int i : nums ){
            set.add(i);
        }
        k=set.size();
        Integer[] uniqueNums = set.toArray(new Integer[0]);
        for (int i = 0; i < uniqueNums.length; i++) {
            nums[i] = uniqueNums[i];
        }
        return k;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int> myset;
        int j = 0;
        for(int i=0; i<nums.size(); i++){
            myset.insert(nums[i]);
        }
        for(int element : myset){
            nums[j] = element;
            j++;
        }
        return myset.size();
    }
};